tests/std/tests/GH_004275_seeking_fancy_iterators/test.cpp

```diff
@@ -0,0 +1,454 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// some portions of this file are derived from libc++'s test files:
+// * support/min_allocator.h

```

Comment:
No change requested: Note that `tests/std` can include headers from libcxx's `support` directory. (We've arranged for this to work in both the GitHub and MSVC-internal test harnesses.) It looks like `<min_allocator.h>` defines a bunch of other stuff that we don't need here, so distilling it down to what we need is reasonable, I just wanted to mention this if it's ever useful in the future.